bin/visual-regression-test.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Color Variables
+BUILD_DIR=$(pwd)
```

Comment:
An enhancement (unrelated to this PR) would be to move all this reused bash stuff into a helper file that is always sourced